### PR TITLE
Created security policy

### PR DESCRIPTION
**Pull request by Kowlin**
Note, this is a draft, done without spell checker. Can't wait for the 12 million comments telling me to fix the grammar :D

### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Supported versions include Red v3 and higher. Red v2 and lower are end of life. And will not receive security updates of any kind. If you are using v2 we recommend you upgrade to v3.
```
Comment (Vexed01) [inline SECURITY.md:5]: This doesn't make sense. Red V2 is not the definition of End of Life.
```suggestion
Supported versions include Red v3 and higher. Red v2 and lower are at end of life. And will not receive security updates of any kind. If you are using v2 we recommend you upgrade to v3.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Examples of acceptable vulnerabilities:
+* An exploit or bug whereby the token is leaked.
+* An exploit or bug whereby data that's stored on the bot is leaked.
+* An exploit whereby it crashes the bot.
+
+If you're reporting a vulnerability for a 3rd party add-on/cog please try to get in contact with its creator. To help you find the creator you can use the bot command ``findcog``.
+
+We will investigate the reported vulnerability and report back on its status when we're able to do so. If the vulnerability is accepted we will open a new security advisory on GitHub.
+We may invite you as the reporter to collaberate on a fix with us if its deemed necessary.
```
Comment (Vexed01) [inline SECURITY.md]: typo
```suggestion
We may invite you as the reporter to collaborate on a fix with us if its deemed necessary. 
```

File: SECURITY.md (hunk 1 above)
Comment (Vexed01) [inline SECURITY.md:5]: For consistency with ``[p]help``'s credits, we should probably capitalise V2 and V3.
```suggestion
Supported versions include Red V3 and higher. Red V2 and lower are end of life. And will not receive security updates of any kind. If you are using V2 we recommend you upgrade to V3.
```

**Review by Vexed01 — COMMENTED** [review]
Not too much to say at this stage.

File: SECURITY.md (hunk 1 above)
Comment (Drapersniper) [reply SECURITY.md:5]: Kinda disagree here, 
`Red v2 and lower reached end-of-life and will not receive security update or fixes of any kind...` would be what makes sense here.

Comment (Vexed01) [reply SECURITY.md:5]: Have reached maybe?

Comment (Drapersniper) [reply SECURITY.md:5]: `Have reached` would be Present Perfect Tense, and `reached` is Simple Past Tense, due to the fact that they reached EOL prior to the release of this PR, simple past tense would be the appropriate tense to use here

Comment (Kowlin) [reply SECURITY.md:5]: I'm not worried about this one. Especially due the fact that this documentation is aimed at people with a technical understanding of security exploits and so forth. Having this parallel with ``help`` is not a big deal.

Comment (mikeshardmind) [reply SECURITY.md:5]: Our minimum supported version shoud be 3.1.8 due to the discord.py fix for application crash with animated emoji

**Pull request closed without merging**